joomla/joomla-cms
# Rename #__history to #__content_history

### Summary of Changes

Rename table `#__history` to `#__content_history`.

Table name `#__history` is too common, besides this table is closely related to content.

Original Joomla 3 table `#__ucm_history` was renamed to `#__history` and it's really weird.

Let's add more logic into table names while it's not too late for Joomla 4.

### Testing Instructions

Install Joomla 4 beta (or update core files in existing installation). Ensure that content history works.

### Actual result BEFORE applying this Pull Request

Table `#__history` exists.

### Expected result AFTER applying this Pull Request

Table `#__history` renamed to `#__content_history`.

### Documentation Changes Required

No.
